modules/api/src/main/java/org/apache/fluo/api/client/FluoClient.java

```diff
@@ -75,6 +77,10 @@ public interface FluoClient extends AutoCloseable {
    */
   MetricsReporter getMetricsReporter();
 
+  default FluoClient useAuthorizations(Collection<String> authLabels) {
+    return this;
+  }
+
```

Comment:
If authorizations can be set in the FluoConfiguration, then this method is probably not needed.  FluoClients are always created from configuration and if the configuration has auths then the client can get them that way.